.gitignore

```diff
@@ -50,3 +50,8 @@ profile.cov
 
 # Generated by tests, and will persist if tests are interrupted.
 les/transactions.rlp
+
+# Files generated by docs build
+/MANIFEST
+/manifest.json
+/site
```

Comment:
What's the deal with these? It seems they're not required for the site... but they get generated anyways? Are they artifacts from some specific plugin?

Reply:
(Asking in part because my first experiment with MkDocs didn't seem to generate them, here https://github.com/meowsbits/ci-tests/tree/gh-pages).

Reply:
Oh... are they artifacts from development?

Reply:
Yes man, they are artifacts from development or custom builds.